package.json

```diff
@@ -10,13 +10,14 @@
     "prop-types": "^15.7.2",
```

Comment:
## OPTIONAL COMMENT:
- [ ] It would be better to fix and solve the ESLint and StyleLint errors before submitting code reviews, I was about to invalidate it and then I saw you fix it. Let us prevent this in the future :muscle: